func_test.go

```diff
@@ -970,7 +970,7 @@ func TestFuncCall(t *testing.T) {
 					}
 
 					// Generate an int with the subtype of the string value
-					outputSet, err := NewValueSet([]Value{Value{
+					outputSet, err := NewValueSet([]Value{{
```

Comment:
Formatting fix due to `gofmt`. We don't need the type of the array element again when instantiating the array.